transmission_interface/include/transmission_interface/differential_transmission.h

```diff
@@ -123,7 +123,8 @@ class DifferentialTransmission : public Transmission
    * \param joint_offset       Joint position offset used in the position mappings.
    * \pre Nonzero actuator and joint reduction values.
    */
-  DifferentialTransmission(const std::vector<double>& actuator_reduction,
+  DifferentialTransmission(const bool ignore_transmission_for_absolute_encoders,
+                           const std::vector<double>& actuator_reduction,
                            const std::vector<double>& joint_reduction,
                            const std::vector<double>& joint_offset = std::vector<double>(2, 0.0));
```

Comment:
Looking at this again, if this is targetting Melodic then one of the two changes should be made to avoid breaking changes:

 - Using delegating constructors, add another constructor with the old signature, and a default value `false` for `ignore_transmission_for_absolute_encoders`
 - Make `ignore_transmission_for_absolute_encoders` the last parameter, with default value `false`

Reply:
I reworked this part, thanks for the suggestion. Now the tests don't need modifications anymore, they test the default value setting of not ignoring. We can add specific tests for the opposite later.

Reply:
:+1: